lib/eex/lib/eex/engine.ex

```diff
@@ -96,6 +96,9 @@ defmodule EEx.Engine do
 
   All other markers are not implemented by this engine.
   """
+  @spec handle_expr(Macro.t, '' | '=', Macro.t) :: Macro.t
```

Comment:
Those are not the only two options allowed. Those are the two options the default handler handles. Just say it assumes a char list. :)